src/App.tsx

```diff
@@ -1,6 +1,7 @@
 import { useTranslation } from 'react-i18next';
 import NotFound from './components/NotFound/NotFound';
```

Comment:
add empty line

Reply:
![image](https://github.com/Igor-Klein-1988/allcom/assets/133020063/0166441c-f25f-4b53-8425-e77d4895f7b7)
like this?